core/src/Cuda/Kokkos_Cuda_Parallel.hpp

```diff
@@ -69,6 +69,25 @@
 
 #include <KokkosExp_MDRangePolicy.hpp>
 
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_SERIAL_EXECUTION
+#ifndef KOKKOS_IMPL_DEFINED_CUDA_USE_SERIAL_SCAN_V
+extern bool kokkos_impl_cuda_use_serial_execution_v;
```

Comment:
Same question here I asked in the other PR - does this have potential to cause any issues with rdc builds?

Reply:
no it shouldn't since this is a host only external variable.

Reply:
wouldn't rdc be safer??  in non-rdc you have more than one copy of the var...

Reply:
uhm again this isn't a device variable. RDC has no impact on this thing whatsoever.